src/gtk/toga_gtk/widgets/label.py

```diff
@@ -2,8 +2,9 @@
 from travertino.size import at_least
 
 from toga.constants import *
```

Comment:
At column 1: (F403) 'from toga.constants import *' used; unable to detect undefined names